app/assets/javascripts/contacts.js

```diff
@@ -4,18 +4,48 @@ var contentType = "application/json";
 var url = "http://localhost:5100/api/contacts";
 var storedContacts = {};
 
+var hideFeedback = function() {
+  $("[data-role='feedback']").hide();
+  $("[data-role='feedback']").remove("p");
+  $("[data-role='feedback']").removeClass("flash-success flash-error");
+};
+
+var showError = function(method) {
+  $("[data-role='feedback']").addClass("flash-error");
+  $("[data-role='feedback']").html(
+    "<p>Error: " + method + " was unsuccessful.</p>"
+  );
+  $("[data-role='feedback']").show();
+};
+
+var showSuccess = function(contact, method) {
```

Comment:
It looks like this function and `showError` do a lot of the same things (add a paragraph tag, add a class, show the feedback role), how about abstracting some of that duplicated logic?